config/locales/en.yml

```diff
@@ -1795,7 +1795,8 @@ en:
       redraft: 'Redraft this claim'
       archive: Archive
       unarchive: Unarchive
-
+    downtime_warning:
+      banner_message: This service will be unavailable between 5:30pm and 11:59pm on %{date_string}
```

Comment:
Based on prior discussions, I thought we were going to wrap values within double quotes?